Assets/Scripts/System/SceneLoadSaveSystem.cs

```diff
@@ -240,7 +242,7 @@ public void MakeEntity(DclScene scene)
                 }
 
 
-                var dclEntity = new DclEntity(guid, customName, parentGuid ?? default, isExposed);
+                var dclEntity = new DclEntity(guid, customName, parentGuid ?? default, isExposed, hierarchyOrder);
```

Comment:
Ah I did this because I wanted to instantiate the entities of the alpha version with their existing hierarchy order values. Should I assign those values afterwards instead ?